feelslikemonday/app/src/main/java/com/example/feelslikemonday/model/MoodType.java

```diff
@@ -0,0 +1,19 @@
+package com.example.feelslikemonday.model;
+
+public class MoodType {
+    private String name;
+    private String emoji;
+
+    public MoodType(String name, String emoji) {
+        this.name = name;
+        this.emoji = emoji;
+    }
+
```

Comment:
why no setters?

Reply:
Don't need to change them to dynamically